src/pages/patientView/mutation/column/AlleleFreqColumnFormatter.tsx

```diff
@@ -31,12 +32,30 @@ export default class AlleleFreqColumnFormatter {
 
             const bar = (<rect x={barX[mutation.sampleId]} y={barY} width={AlleleFreqColumnFormatter.barWidth} height={barHeight} fill={sampleManager.getColorForSample(mutation.sampleId)}/>);
 
-            const circleRadius = 6;
             const sampleId = mutation.sampleId;
-            const component = sampleManager.getComponentForSample(sampleId);
+            const isUncalled = mutation.geneticProfileId.endsWith(GENETIC_PROFILE_UNCALLED_MUTATIONS_SUFFIX);
+            let opacity: number = 1;
+            let extraTooltipText: string = '';
+            if (isUncalled) {
+                if (altReads > 0) {
+                    opacity = 0.1;
+                    extraTooltipText = "Mutation has supporting reads, but wasn't called";
+                } else {
+                    opacity = 0;
+                    extraTooltipText = "Mutation has 0 supporting reads and wasn't called";
+                }
+            }
+            const component = sampleManager.getComponentForSample(sampleId, opacity, extraTooltipText);
+
+            let variantReadText:string;
+            if (isUncalled && altReads > 0) {
+                variantReadText = `(${altReads} variant reads out of ${altReads+refReads} total, but uncalled)`;
+            } else {
+                variantReadText = `(${altReads} variant reads out of ${altReads+refReads} total)`;
```

Comment:
As mentioned on the chat, it's to show that position was profiled. Otherwise you don't know whether or not that position was actually checked for read counts.